lib/export/formatters/dataElementFormatterCimpl6.js

```diff
@@ -31,7 +31,6 @@ class DataElementFormatterCimpl6 {
   set maxPlacement(maxPlacement) { this._maxPlacement = maxPlacement; }
   get maxRightPad() { return this._maxRightPad; }
   set maxRightPad(maxRightPad) { this._maxRightPad = maxRightPad; }
-  get substituteHash() { return this._substituteHash};
```

Comment:
Just to be clear, I'm assuming this was dead code (i.e., not used anywhere), right?

Reply:
Yes, this was dead code that should have been removed before. It was not used anywhere else and was originally placed there to help with the substitutions being made. However, I ended up going a different direction with it.